src/components/Header/Header.module.css

```diff
@@ -5,36 +5,74 @@
 \*------------------------------------*/
 
 /**
- * 1) 
+ * 1) Global container for the header component. This helps set the stage for header recipes across applications
  */
 .header {
+  display: flex;
   background-color: var(--eds-theme-color-primary-background);
   color: var(--eds-theme-color-body-foreground-inverted);
-  padding: var(--eds-size-2);
+  padding: var(--eds-size-1) var(--eds-size-2);
   position: relative;
+  width: 100%;
+  z-index: var(--eds-z-index-500);
 
   @media all and (min-width: $eds-bp-lg) {
-    padding: var(--eds-size-6) var(--eds-size-4) var(--eds-size-4)
-      var(--eds-size-4);
+    flex-direction: column;
+    width: auto;
+    padding: var(--eds-size-6) var(--eds-size-2-and-half) var(--eds-size-6)
+      var(--eds-size-2-and-half);
     height: 100vh;
   }
 }
 
-.header__inner {
-  display: flex;
-  align-items: center;
-  gap: var(--eds-size-2);
+/**
+ * Container around sticky header
+ * 1) This is added to
+ */
+.header-sticky-container {
+  height: 100vh;
+  transition: all var(--eds-anim-move-quick) var(--eds-anim-ease);
```

Comment:
Should we be transitioning `all`? Could we target the specific properties instead?